tests/bluetooth/tester/src/bttester.h

```diff
@@ -228,6 +229,17 @@ struct gap_passkey_confirm_cmd {
 	u8_t match;
 } __packed;
 
+#define GAP_START_DIRECTED_ADV		0x15
```

Comment:
Not related to this PR: I know this file is already screwed up from the beginning, but in the long run we really should stop using tabs for alignment, and use spaces instead. This makes it possible to verify the proper alignment in any kind of UI (including the web browser). Tabs should only exist in the very beginning of a line, i.e. only be used for indentation.

Reply:
Yes, I added a comment about this. The original file was using tabs so I did not change that in the update.